scipy/sparse/linalg/isolve/iterative.py

```diff
@@ -145,7 +145,8 @@ def bicg(A, b, x0=None, tol=1e-5, maxiter=None, xtype=None, M=None, callback=Non
 
 @set_docstring('Use BIConjugate Gradient STABilized iteration to solve A x = b',
                'The real or complex N-by-N matrix of the linear system\n'
-               '``A`` must represent a hermitian, positive definite matrix')
+               'It is required that the linear operator can produce\n'
```

Comment:
Tiny complaint: should add a period after "linear system"; otherwise the sentences run together in the docs:

https://1332-1460385-gh.circle-artifacts.com/0/home/ubuntu/scipy/doc/build/html-scipyorg/generated/scipy.sparse.linalg.bicgstab.html#scipy.sparse.linalg.bicgstab

Presumably this resulted from studying the docstring for `bicg`, which unfortunately has the same defect. Mind fixing that too?